libs/docs/shared/src/lib/core-helpers/toolbar/toolbar.component.ts

```diff
@@ -211,6 +191,21 @@ export class ToolbarDocsComponent implements OnInit, OnDestroy {
         this._contentDensityService.updateContentDensity(density);
     }
 
+    private _setVersions(): void {
+        this._http
+            .get<Version[]>('https://raw.githubusercontent.com/SAP/fundamental-ngx/main/versions.json')
```

Comment:
Do we need to actually call a http request? File is in repo, and can be imported directly

Reply:
Here is the trick, actually :) 

When new version will be released it will be added to the file, so the old deployments can show links to the newer versions.

Reply:
Otherwise `v0.37` won't get `v0.51` deployment's url because at the moment of building the old one its `versions.json` has any versions not newer than `v0.37`.

Reply:
Got it, thanks